orchagent/muxorch.cpp

```diff
@@ -841,8 +928,44 @@ void MuxOrch::updateNeighbor(const NeighborUpdate& update)
         if (ptr->isIpInSubnet(update.entry.ip_address))
         {
             ptr->updateNeighbor(update.entry, update.add);
+            return;
         }
     }
+
+    string port, old_port;
+    if (update.add && !getMuxPort(update.mac, update.entry.alias, port))
+    {
+        return;
+    }
+    else if (update.add)
+    {
+        /* Check if the neighbor already exists */
+        old_port = getNexthopMuxName(update.entry);
+        addNexthop(update.entry);
+    }
+    else
```

Comment:
Assuming there are no other update type and when reached here it must be the delete case?

Reply:
yes